src/Core/src/Platform/iOS/MauiTimePicker.cs

```diff
@@ -47,15 +44,50 @@ public MauiTimePicker()
 			InputAssistantItem.TrailingBarButtonGroups = null;
 
 			AccessibilityTraits = UIAccessibilityTrait.Button;
+
+			EditingDidBegin += OnEditingDidBegin;
+			EditingDidEnd += OnEditingDidEnd;
+			ValueChanged += OnValueChanged;
+			_picker.ValueChanged += OnValueChanged;
```

Comment:
So no need to unsubscribe these?

Reply:
Because there is no lifecycle event to reliably unsubscribe. The fix here is to make it where you don't even need to. If the `MauiTimePicker` type subscribes to its own events, it just holds extra references to itself. That won't cause a leak.

@PureWeen is working on a new callback for handlers. If it works out, maybe we won't have to do this.

For now, I'm going to split out the `MauiAccessoryView` fix in here and do a new PR for that part.